src/main/java/com/iexec/blockchain/tool/IexecHubService.java

```diff
@@ -166,7 +166,7 @@ public boolean isTaskInUnsetStatusOnChain(String chainTaskId) {
         return getChainTask(chainTaskId)
```

Comment:
Why not
return getChainTask(chainTaskId).isEmpty()
?
(note: neither your suggestion nor mine handles the issue where the node doesn't reply, we should handle such exception elsewhere in the future)